packages/common/src/types/common.ts

```diff
@@ -2,7 +2,7 @@ import { Request } from 'express';
 import { ObjectId } from 'mongodb';
 
 type TDbRecordBasic = {
-  _id: ObjectId;
+  _id?: ObjectId;
```

Comment:
Dont touch the basic record predefined

Reply:
How to insert object then?

Reply:
fixed

Reply:
@magestrio When you defined type of Model you should use `ModelBasic<WithoutId<TRollUpHistoryRecord>>` to prevent include `_id`

Reply:
already included